### ISSUE TITLE
why not load service's meta

### ISSUE THREAD

**Issue by klj890** [hashicorp/consul#4533]
Issues on GitHub are intended to be related to bugs or feature requests, so we recommend using our other community resources instead of asking here.

- [FAQ (Frequently asked questions)](https://www.consul.io/docs/faq.html)
- [Consul Guides](https://www.consul.io/docs/guides/index.html)
- Any other questions can be sent to the [consul mailing list](https://www.consul.io/community.html)

this my config

```
{
  "services":[
     {
       "id":"qcmm_datagate_1",
       "name":"qcmm_datagete_v3",
       "tags":["qcmm", "V3", "UAT", "datagate"],
       "port": 8800,
       "address" : "192.168.182.10",
       "meta":{
         "protocol":"http",
         "path":"/datagate/api/"
       }
     },
     {
       "id":"qcmm_datagate_2",
       "name":"qcmm_datagete_v3",
       "tags":["qcmm", "V3", "UAT", "datagate"],
       "port": 8800,
       "address" : "192.168.182.10",
       "meta":{
         "protocol":"http",
         "path":"/datagate/api/"
       }
     }
   ]
}
```

and, this api response：
```
[
   {
       "ID":"66c9d2a5-586c-44d0-65d2-9c403917a8a0",
       "Node":"i-e9bz9fgd",
       "Address":"192.168.182.10",
        "Datacenter":"qcmm",
       "TaggedAddresses":{"lan":"192.168.182.10","wan":"192.168.182.10"},
       "NodeMeta":{"consul-network-segment":""},
   "ServiceKind":"",
"ServiceID":"qcmm_datagate_1",
"ServiceName":"qcmm_datagete_v3",
"ServiceTags":["qcmm","V3","UAT","datagate"],
"ServiceAddress":"192.168.182.10",
"ServiceMeta":null,
"ServicePort":8800,
"ServiceEnableTagOverride":false,
"ServiceProxyDestination":"",
"ServiceConnect":{"Native":false,"Proxy":null},
"CreateIndex":2786746,
"ModifyIndex":2786746},
{"ID":"66c9d2a5-586c-44d0-65d2-9c403917a8a0","Node":"i-e9bz9fgd","Address":"192.168.182.10","Datacenter":"qcmm","TaggedAddresses":{"lan":"192.168.182.10","wan":"192.168.182.10"},"NodeMeta":{"consul-network-segment":""},"ServiceKind":"","ServiceID":"qcmm_datagate_2","ServiceName":"qcmm_datagete_v3","ServiceTags":["qcmm","V3","UAT","datagate"],"ServiceAddress":"192.168.182.10","ServiceMeta":null,"ServicePort":8800,"ServiceEnableTagOverride":false,"ServiceProxyDestination":"","ServiceConnect":{"Native":false,"Proxy":null},"CreateIndex":2786747,"ModifyIndex":2786747}]
```
why ServiceMeta is null?

**Comment by klj890** [comment]
server is Consul v1.0.3
client node is  Consul v1.2.2

thanks

**Comment by klj890** [comment]
yes!
I update server to v1.2.2, and is OK.
Thanks!!!!!! @pierresouchay

**Comment by pierresouchay** [comment]
@klj890 yes, this feature is available from 1.0.7+
